XamlControlsGallery/Common/ThemeHelper.cs

```diff
@@ -13,7 +13,7 @@ public static class ThemeHelper
     {
         private const string SelectedAppThemeKey = "SelectedAppTheme";
         private static Window CurrentApplicationWindow;
-        // Keep reference so it does not optimized/gc'ed away
+        // Keep reference so it does not get optimized/garbage collected
```

Comment:
nit²: I think "optimized" may be a bit confusing, and "optimized away" may be a bit more descriptive. However either way it's clear so I guess I am a bit nit picky here.

Reply:
Only made this change as my spellchecker picked up "gc'ed". Not particularly important.